### ISSUE TITLE
Too many humans c contributing!

### ISSUE THREAD

**Issue by draculaphd** [bradfordcondon/SWC_UTK_git#13]
I find hte caffeine monster definition to be too loose.  

**I DEMAND MONSTERS ONLY CONTRIBUTIONS**

**Comment by rwmurdoch** [comment]
<- plot monster

**Comment by draculaphd** [comment]
#12  I HAVENT LOOKED AT THIS BUT IM SURE ITS HUMAN NONSENSE

**Comment by joseliquetgonzalez** [comment]
burritos rulez

**Comment by svaranas123** [comment]
can Draq tolerate pluto?

**Comment by draculaphd** [comment]
Is pluto a planet?  This probably needs to be its own issue for discussion

**Comment by svaranas123** [comment]
True! Let’s call it a dwarf planet. If that’s where we can send the draq dude.

Sent from my iPhone

On Dec 19, 2017, at 5:34 PM, draculaphd <notifications@github.com<mailto:notifications@github.com>> wrote:


Is pluto a planet? This probably needs to be its own issue for discussion
